docs/reference/commandline/attach.md

```diff
@@ -14,6 +14,7 @@ parent = "smn_cli"
 
     Attach to a running container
 
+      --detach-keys       Specify the escape key sequence used to detach a container
```

Comment:
@vdemeester Nice feature. I think this needs a bit of rearranging and more info on the flag: 

---> https://gist.github.com/moxiegirl/b20a296942ab31d62e56

# attach

    Usage: docker attach [OPTIONS] CONTAINER

    Attach to a running container

      --detach-keys="ctrl-<value>"       Set up escape key sequence
      --help              Print usage
      --no-stdin          Do not attach STDIN
      --sig-proxy=true    Proxy all received signals to the process

The `docker attach` command allows you to attach to a running container using
the container's ID or name, either to view its ongoing output or to control it
interactively. You can attach to the same contained process multiple times
simultaneously, screen sharing style, or quickly view the progress of your
detached  process.

To stop a container, use `CTRL-c`. This key sequence sends `SIGKILL` to the
container. If `--sig-proxy` is true (the default),`CTRL-c` sends a `SIGINT` to
the container. You can detach from a container and leave it running using the
using `CTRL-p CTRL-q` key sequence.

> **Note:**
> A process running as PID 1 inside a container is treated specially by
> Linux: it ignores any signal with the default action. So, the process
> will not terminate on `SIGINT` or `SIGTERM` unless it is coded to do
> so.

It is forbidden to redirect the standard input of a `docker attach` command
while attaching to a tty-enabled container (i.e.: launched with `-t`).

## Override the detach sequence

If you want, you can configure a different sequence for detach. Configuration is
useful if the Docker default sequence overrides a default you use for other
applications. There are two ways to configure a custom detach key sequence, per
container or on your entire configuration.

To change the sequence for a particular container, use the
`--detach-keys="ctrl-<value>"` flag with the `docker attach` command. Replace
`<value>` with any of the following characters:

* `a-z` (a single lowercase alpha character )
* `@` (ampersand)
* `[` (left bracket)
* `\\` (two backward slashes)
*  `_` (underscore)
* `^` (caret)

To configure a different key sequence for all containers, see [**Configuration
file** section](cli.md#configuration-files).